nav2_system_tests/src/planning/test_planner_costmaps_node.cpp

```diff
@@ -15,6 +15,7 @@
 #include <gtest/gtest.h>
 #include <memory>
 #include <vector>
+#include <iostream>
```

Comment:
I think we can remove this include.

Reply:
This was a suggestion from ament's linting tools

Reply:
done